app/assets/stylesheets/partials/_event_box.scss

```diff
@@ -1,45 +1,57 @@
 .event-box {
+  @include media(medium) { padding: 1rem 1.3rem; }
+  @include media(small) { padding: .4rem .5rem; }
+  
   padding: 2.1rem 1.8rem;
   border: 1px solid $v3-color_green-light;
   overflow: hidden;
 
-  @include media(medium) { padding: 1rem 1.3rem; }
-
   &__header {
+    @include media(medium) { justify-content: center; }
+
     display: flex;
     width: 100%;
 
     h2 {
```

Comment:
While we're here might we be able to also refactor this to follow BEM a little more closely by replacing `h2` with a class selector here and in the DOM?

Reply:
Good catch---I updated `_event_box.html.erb` so that the h2 had the class name `.event-box__header` as opposed to being the only child of a div with the same class name. I updated the styling in `_event_box.scss` to reflect this change. Still taking a look at the media queries/mixins.